### PR TITLE
Services article

### PR DESCRIPTION
**Pull request by sbastn**
Based on #128, this PR adds a services article.

I renamed the current "adding services" to "contributing services" and used "adding services" for this content. The reason why is "adding services" is because I want it to be the top article on the service category because I think it's the most relevant to most of our customers (articles are sorted by name).

### CODE REVIEW

File: content/articles/adding-services.markdown [hunk 1]
```diff
@@ -5,8 +5,49 @@ categories:
...
+* Amazon Web Services
+* Cloudflare
+* GitHub pages
+* Google Apps
+* Heroku
+* Mailgun
+* Shopify
+* Wordpress
+* Squarespace
+* Windows Azure
```
Comment (weppos) [inline content/articles/adding-services.markdown]: I'd add an `<info>` tag here that explains

> The service directory is open-source. Learn [how to propose or contribute new services](/articles/contributing-service).

**Comment by weppos** [comment]
As this is now a generic article about services, I'd rename `adding-services` to `services`.

**Comment by sbastn** [comment]
> As this is now a generic article about services, I'd rename adding-services to services.

Then it will show up at the bottom of this page: http://support.dnsimple.com/categories/services/ and that sucks.

**Comment by weppos** [comment]
The order is not based on naming. Moreover, you can use the `articles.yaml` file to define a priority. As there are no other services article there, just add `services.markdown` to the end of the file.

File: content/articles/services.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+## What are services
+
+Services are DNS snippets ready for you to use. Applying a service creates all the DNS configuration needed to serve your webpage or configure your email.
+
+We currently support more than 30 services! Here are the most recognized ones:
+
+* Amazon Web Services
+* Cloudflare
+* GitHub pages
+* Google Apps
```
Comment (jcaudle) [inline content/articles/services.markdown]: I'd suggest linking to the article on the Google Apps service: http://support.dnsimple.com/articles/google-apps-service/

File: content/articles/services.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+---
+
+## What are services
+
+Services are DNS snippets ready for you to use. Applying a service creates all the DNS configuration needed to serve your webpage or configure your email.
+
+We currently support more than 30 services! Here are the most recognized ones:
+
+* Amazon Web Services
+* Cloudflare
```
Comment (jcaudle) [inline content/articles/services.markdown]: Similarly, I'd suggest linking to the Cloudflare service page: http://support.dnsimple.com/articles/cloudflare-service/.

**Comment by jcaudle** [comment]
I like the article on generally using services a lot. This should be useful for customers. :100:

**Pull request merged**